### DISCUSSION TITLE
Download GIF files

### DISCUSSION THREAD

**Discussion by billbobm** [Dineshkarthik/telegram_media_downloader#166] [Ideas]
**Is your feature request related to a problem? Please describe.**
When downloading photos, GIF files are not downloaded.

**Describe the solution you'd like**
Either include GIF under the all flag, or add it as specific file format under the photo category

**Comment by Dineshkarthik** [comment]
Hi,

Usually the gif you sent from sticker search are consider as stickers. So if those gifs need to be downloaded then the feature to download sticker needs to be added. Converting this into a discussion to see if any other considerable number of users need this feature.

**Comment by Quackdoc** [comment]
+1 also I think part of this would be adding Animation. although those are mostly converted to mp4s, the "animations" are their own distinct thing, and are often treated as gifs

**Comment by dzwzqlx** [comment]
Dear author:

I need this feature too. 

By the way, the GIFs in my group was send from some screenshort tool, or downloaded from web, not "sent from sticker search", I think they were stored in the telegram server as MP4s, if the python tool will download them as MP4s, that's ok too, because I can transform them easily with ffmpeg.

If this feature will not be included in the future, could you please give me a hint of how to edit the source code to get it? I can give a try in my own repository. Thank you very much!

**Comment by mbarreradev** [comment]
It doesn't make sense that you can download every type of file except a gif, yes a gif in 2023

**Comment by Durjay-Dhali** [comment]
please add a option to download gifs. thank you
